content/pages/about.md

```diff
@@ -1,6 +1,9 @@
 ---
 title: About 18F
 permalink: /about/
+eleventyNavigation:
```

Comment:
clarification: It's starting to look like we have two different methods for setting navigation info — the navigation data, and now this. Is that perception correct?

Reply:
Stopped using the 11ty nav plugin. Should we uninstall the plugin?
Created a nav collection from a cleaned up _data/navigation.yml and built the nav from that.